Riot/Assets/en.lproj/Untranslated.strings

```diff
@@ -19,3 +19,4 @@
 // MARK: Onboarding Personalization WIP
 
 "image_picker_action_files" = "Choose from files";
+"voice_broadcast_in_timeline" = "Voice broadcast detected (under active development). We currently only detect voice broadcast in the room timeline, this is not possible to send or listen an actual voice broadcast.";
```

Comment:
I would remove the point at the end of the string. The other Labs labels don't have point